pages/common/git-ls-files.md

```diff
@@ -0,0 +1,16 @@
+# git ls-files
+
+> Git command for showing information about files in the index and the working tree.
```

Comment:
Perhaps this will read better?
```suggestion
> Show information about files in the index and the working tree.
```

Reply:
@sbrl I do agree that this is better, however notice that some git commands have descriptions like this, and this is where i get this template from :)